src/state/wallets/selectors.ts

```diff
@@ -97,7 +97,7 @@ export const transactions = createSelector(wallets, electrumXSelectors.blockHeig
         let toInternalAddress;
 
         const toAddress = transaction.outputs[0].addresses[0];
-        const isToInternalAddress = wallet.weOwnAddress(toAddress);
+        const isToInternalAddress = walletsList.some(w => w.weOwnAddress(toAddress));
```

Comment:
is not touch other code ?

Reply:
what do you mean? this is PR to the previous PR about the transaction list

Reply:
Just question is = isToInternalAddress is not used elsewhere ? if not is ok no more questions

Reply:
no, it is not